packages/lib/server/service/team.ts

```diff
@@ -120,4 +123,56 @@ export class TeamService {
     const teamBilling = await TeamBilling.findAndInit(teamId);
     return teamBilling.publish();
   }
+
+  static async getTeamWithEventTypes(teamSlug: string, meetingSlug: string, orgSlug: string | null) {
+    const team = await prisma.team.findFirst({
+      where: {
+        ...getSlugOrRequestedSlug(teamSlug),
+        parent: orgSlug ? getSlugOrRequestedSlug(orgSlug) : null,
+      },
+      orderBy: {
+        slug: { sort: "asc", nulls: "last" },
+      },
+      select: {
+        id: true,
+        isPrivate: true,
+        hideBranding: true,
+        parent: {
+          select: {
+            id: true,
+            slug: true,
+            name: true,
+            bannerUrl: true,
+            logoUrl: true,
+            hideBranding: true,
+            organizationSettings: {
+              select: {
+                allowSEOIndexing: true,
+              },
+            },
+          },
+        },
+        logoUrl: true,
+        name: true,
+        slug: true,
+        brandColor: true,
+        darkBrandColor: true,
+        theme: true,
+        eventTypes: {
+          where: {
+            OR: [{ slug: meetingSlug }, { slug: { startsWith: `${meetingSlug}-team-id-` } }],
+          },
+          select: getPublicEventSelect(false),
```

Comment:
This is a very important line. This is so that `team.eventTypes[0]` (used for event data in team booking page) has everything expected in Booker (which used to rely on `getPublicEventSelect` trpc call)